README.md

```diff
@@ -0,0 +1,24 @@
+# SDCTA-Data-Visualization-Website
+Install dependencies then run the backend:
+```bash
+$ cd server
+$ npm install
+$ npm start
+```
+
+In a separate terminal install dependencies and run the frontend:
+```bash
+$ cd SDCTA
+$ npm install
+$ npm start
+```
+Note: You will need a .env file that defines DATABASE_URL.
+
+(Currently in progress)
```

Comment:
If this is in progress, why not just remove this and put an issue up about it?

Reply:
Because its essentially done, going to put the PR up soon.

Reply:
Removed it, will just add it with that PR